docs/output.md

```diff
@@ -142,6 +174,28 @@ Filename abbreviations stand for the following reference alignment strands:
 * `sample.bedGraph`
   * Methylation statuses in [bedGraph](http://genome.ucsc.edu/goldenPath/help/bedgraph.html) format.
 
+**biscuit workflow output directory: `results/methylation_extract/`**
+
+* `sample.bedgraph`
+  * Methylation statuses in [bedGraph](http://genome.ucsc.edu/goldenPath/help/bedgraph.html) format.
+* `sample.bedgraph`
+  * Methylation statuses in [bedGraph](http://genome.ucsc.edu/goldenPath/help/bedgraph.html) format, intersected with soloWCGW file.
+  *  **NB:** Created only if `--soloWCGW_file` is specified. **EXPERIMENTAL!**
+ * `sample.vcf.gz` 
+	 * VCF file with the pileup information, used for creating the bedgraph file.
+	 *  **NB:** Only saved if `--save_pileup_file` is specified when running the pipeline.
+ * `sample.vcf.gz.tbi` 
+	 * Index file for `sample.vcf.gz`
+	 *  **NB:** Only saved if `--save_pileup_file` is specified when running the pipeline.
+> **NB** if `--epriread` is specified in the pipeline, then:
+> **output directory:** `results/epireads` :
+	> * `sample.epiread` 
+			 Storing CpG retention pattern as well as SNP information on the same read in a compact way 
+> * `snp/sample.snp.bed`
+	> 	Storing the SNP information of the `sample.epiread` file. **EXTRACTING SNP PROCSS IS IN PROGRESS!**
```

Comment:
Is this something you want to add to this PR?

Reply:
Hmm.. I'll remove that until the biscuit team finish to work on it...